<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="0" hostname="DESKTOP-9J129ML" name="org.example.testRunner.runners" tests="19" failures="2" timestamp="2022-07-27T21:51:49 EET" time="340.524" errors="1">
  <testcase name="runScenario" time="22.224" classname="org.example.testRunner.runners"/>
  <system-out/>
  <testcase name="runScenario" time="12.382" classname="org.example.testRunner.runners"/>
  <system-out/>
  <testcase name="runScenario" time="16.281" classname="org.example.testRunner.runners"/>
  <system-out/>
  <testcase name="runScenario" time="17.969" classname="org.example.testRunner.runners"/>
  <system-out/>
  <testcase name="runScenario" time="23.549" classname="org.example.testRunner.runners"/>
  <system-out/>
  <testcase name="runScenario" time="17.604" classname="org.example.testRunner.runners"/>
  <system-out/>
  <testcase name="runScenario" time="20.205" classname="org.example.testRunner.runners"/>
  <system-out/>
  <testcase name="runScenario" time="10.199" classname="org.example.testRunner.runners">
    <failure type="java.lang.AssertionError" message="The following asserts failed:
	expected [https://demo.nopcommerce.com/iphone-6] but found [https://demo.nopcommerce.com/]">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
expected [https://demo.nopcommerce.com/iphone-6] but found [https://demo.nopcommerce.com/]
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:46)
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:30)
at org.example.stepDefs.D06_homeSlidersStepDef.directUserToExpectedSecondSlideUrl(D06_homeSlidersStepDef.java:44)
at ✽.direct user to expected second slide url(file:///C:/apache-maven-3.8.6/demonopcommerce/src/main/resources/features/F06_homeSliders.feature:10)
]]>
    </failure>
  </testcase> <!-- runScenario -->
  <system-out/>
  <testcase name="runScenario" time="15.896" classname="org.example.testRunner.runners"/>
  <system-out/>
  <testcase name="runScenario" time="17.409" classname="org.example.testRunner.runners"/>
  <system-out/>
  <testcase name="runScenario" time="20.006" classname="org.example.testRunner.runners"/>
  <system-out/>
  <testcase name="runScenario" time="30.350" classname="org.example.testRunner.runners"/>
  <system-out/>
  <testcase name="runScenario" time="12.423" classname="org.example.testRunner.runners"/>
  <system-out/>
  <testcase name="runScenario" time="15.698" classname="org.example.testRunner.runners"/>
  <system-out/>
  <testcase name="runScenario" time="10.095" classname="org.example.testRunner.runners">
    <failure type="java.lang.AssertionError" message="The following asserts failed:
	expected [https://demo.nopcommerce.com/nokia-lumia-1020] but found [https://demo.nopcommerce.com/]">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
expected [https://demo.nopcommerce.com/nokia-lumia-1020] but found [https://demo.nopcommerce.com/]
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:46)
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:30)
at org.example.stepDefs.D06_homeSlidersStepDef.directUserToExpectedUrl(D06_homeSlidersStepDef.java:30)
at ✽.direct user to expected url(file:///C:/apache-maven-3.8.6/demonopcommerce/src/main/resources/features/F06_homeSliders.feature:6)
]]>
    </failure>
  </testcase> <!-- runScenario -->
  <system-out/>
  <testcase name="runScenario" time="22.482" classname="org.example.testRunner.runners"/>
  <system-out/>
  <testcase name="runScenario" time="20.354" classname="org.example.testRunner.runners"/>
  <system-out/>
  <testcase name="runScenario" time="21.439" classname="org.example.testRunner.runners"/>
  <system-out/>
  <testcase name="runScenario" time="13.959" classname="org.example.testRunner.runners">
    <error type="java.lang.IndexOutOfBoundsException" message="Index: 1, Size: 1">
      <![CDATA[java.lang.IndexOutOfBoundsException: Index: 1, Size: 1
at java.util.ArrayList.rangeCheck(ArrayList.java:659)
at java.util.ArrayList.get(ArrayList.java:435)
at org.example.stepDefs.D07_followUsStepDef.isOpenedInNewTab(D07_followUsStepDef.java:26)
at ✽."https://demo.nopcommerce.com/new-online-store-is-open" is opened in new tab(file:///C:/apache-maven-3.8.6/demonopcommerce/src/main/resources/features/F07_followUs.feature:14)
]]>
    </error>
  </testcase> <!-- runScenario -->
  <system-out/>
</testsuite> <!-- org.example.testRunner.runners -->
